wigwam/commands.py

```diff
@@ -252,19 +256,85 @@ def cmake_install(tag: str, base: str, no_cache: bool = False) -> Image:
     )
 
 
-def dropin(tag: str) -> None:
+def test(
+    tag: str,
+    output_xml: os.PathLike[str] | str,
+    compress_output: bool,
+    quiet_fail: bool,
+) -> None:
+    """
+    Run all ctests from the Docker image work directory.
+
+    Parameters
+    ----------
+    tag : str
+        The tag of the image to test.
+    output_xml : os.PathLike[str] | str
+        The name of the XML test output file.
+    compress_output : bool
+        If True, compress the output of the test.
+    quiet_fail : bool
+        If True, don't generate verbose output on failure.
+    """
+
+    prefixed_tag: str = prefix_image_tag(tag)
+
+    xml_filename = Path(output_xml).name
+
+    image_volume_path = "/temporary/Testing"
```

Comment:
Why did we change this from `/tmp/Testing` to `/temporary/Testing`?

Reply:
I recall at some point having a discussion about `/tmp` being a directory that Linux does things with - I wanted to avoid any chance of the OS doing anything unusual with the directory while we're using it, so I moved over to `/temporary` which is just an arbitrary directory name.

Reply:
Hmm, the only particular special behavior that I'm aware of is that some OSes will clean up old files in `$TMPDIR` when the system boots.

I'm slightly wary of using `/temporary` instead of `/tmp` without a specific reason, just because I think it'd be confusing for folks reading the code since `/tmp` is the typical conventional location for temporary files.

If you prefer to avoid `/tmp`, how about calling it `/scratch` instead?